diff --git a/app/frontend/shared/entities/ticket-article/action/plugins/types.ts b/app/frontend/shared/entities/ticket-article/action/plugins/types.ts
index d4e3e53f37..f8659e71da 100644
--- a/app/frontend/shared/entities/ticket-article/action/plugins/types.ts
+++ b/app/frontend/shared/entities/ticket-article/action/plugins/types.ts
@@ -32,6 +32,7 @@ export interface TicketArticleFormValues {
   body?: string
   internal?: boolean
   cc?: string[]
+  bcc?: string[]
   subtype?: string
   inReplyTo?: string
   to?: string[]
@@ -100,6 +101,7 @@ export interface TicketArticleTypeProps {
 export interface TicketFieldsType {
   to: TicketArticleTypeProps
   cc: TicketArticleTypeProps
+  bcc: TicketArticleTypeProps
   subject: TicketArticleTypeProps
   body: TicketArticleTypeProps
   attachments: TicketArticleTypeProps
@@ -110,6 +112,7 @@ export interface TicketFieldsType {
 export interface TicketArticleTypeFields {
   to: TicketArticleTypeReactiveFieldProps
   cc: TicketArticleTypeReactiveFieldProps
+  bcc: TicketArticleTypeReactiveFieldProps
   subject: TicketArticleTypeReactiveFieldProps
   body: TicketArticleTypeReactiveFieldProps
   attachments: TicketArticleTypeReactiveFieldProps
