diff --git a/app/frontend/shared/entities/ticket-article/action/plugins/email.ts b/app/frontend/shared/entities/ticket-article/action/plugins/email.ts
index d0eb8a5ca7..530e51a3b4 100644
--- a/app/frontend/shared/entities/ticket-article/action/plugins/email.ts
+++ b/app/frontend/shared/entities/ticket-article/action/plugins/email.ts
@@ -116,6 +116,7 @@ const actionPlugin: TicketArticleActionPlugin = {
     const fields: Partial<TicketFieldsType> = {
       to: { required: true },
       cc: {},
+      bcc: {}, // KC: BCC field
       subject: {},
       body: {
         required: true,
