diff --git a/app/frontend/shared/entities/ticket-article/action/plugins/email.ts b/app/frontend/shared/entities/ticket-article/action/plugins/email.ts
index b7142d7264..cdc50eab77 100644
--- a/app/frontend/shared/entities/ticket-article/action/plugins/email.ts
+++ b/app/frontend/shared/entities/ticket-article/action/plugins/email.ts
@@ -130,6 +130,7 @@ const actionPlugin: TicketArticleActionPlugin = {
     const fields: Partial<TicketFieldsType> = {
       to: { required: true },
       cc: {},
+      bcc: {}, // KC: BCC field
       subject: {},
       body: {
         required: true,
