diff --git a/app/frontend/shared/graphql/types.ts b/app/frontend/shared/graphql/types.ts
index 297527b4e6..91f7304ffa 100644
--- a/app/frontend/shared/graphql/types.ts
+++ b/app/frontend/shared/graphql/types.ts
@@ -4299,6 +4299,8 @@ export type TicketArticleInput = {
   body?: InputMaybe<Scalars['String']['input']>;
   /** The article CC address. */
   cc?: InputMaybe<Array<Scalars['String']['input']>>;
+  /** The article BCC address. */
+  bcc?: InputMaybe<Array<Scalars['String']['input']>>;
   /** The article content type. */
   contentType?: InputMaybe<Scalars['String']['input']>;
   /** The article sender address. */
