diff --git a/app/frontend/shared/graphql/schema-types.ts b/app/frontend/shared/graphql/schema-types.ts
index bc48296203..47003eed2a 100644
--- a/app/frontend/shared/graphql/schema-types.ts
+++ b/app/frontend/shared/graphql/schema-types.ts
@@ -4371,6 +4371,8 @@ export type TicketArticleInput = {
   body?: InputMaybe<Scalars['String']['input']>;
   /** The article CC address. */
   cc?: InputMaybe<Array<Scalars['String']['input']>>;
+  /** The article BCC address. */
+  bcc?: InputMaybe<Array<Scalars['String']['input']>>;
   /** The article content type. */
   contentType?: InputMaybe<Scalars['String']['input']>;
   /** The article sender address. */
