diff --git a/app/frontend/shared/graphql/schema-types.ts b/app/frontend/shared/graphql/schema-types.ts
index 75c1918208..12f9f71e8d 100644
--- a/app/frontend/shared/graphql/schema-types.ts
+++ b/app/frontend/shared/graphql/schema-types.ts
@@ -5117,6 +5117,8 @@ export type TicketArticleInput = {
   body?: InputMaybe<Scalars['String']['input']>;
   /** The article CC address. */
   cc?: InputMaybe<Array<Scalars['String']['input']>>;
+  /** The article BCC address. */
+  bcc?: InputMaybe<Array<Scalars['String']['input']>>;
   /** The article content type. */
   contentType?: InputMaybe<Scalars['String']['input']>;
   /** The article sender address. */
